Source/objects.cpp

```diff
@@ -4891,29 +4801,24 @@ void OperateObject(Player &player, int i, bool teleFlag)
 	}
 }
 
-void DeltaSyncOpObject(int cmd, Object &object)
+void DeltaSyncOpObject(Object &object)
 {
 	switch (object._otype) {
 	case OBJ_L1LDOOR:
 	case OBJ_L1RDOOR:
-		SyncOpL1Door(cmd, object);
-		break;
 	case OBJ_L2LDOOR:
 	case OBJ_L2RDOOR:
-		SyncOpL2Door(cmd, object);
-		break;
 	case OBJ_L3LDOOR:
 	case OBJ_L3RDOOR:
-		SyncOpL3Door(cmd, object);
-		break;
 	case OBJ_L5LDOOR:
 	case OBJ_L5RDOOR:
-		SyncOpL5Door(cmd, object);
+		object._oAnimFrame += 2;

```

Comment:
This is to capture a side-effect of `SyncOpL*Door` that's no longer triggered by the switch to `OpenDoor` right?

Reply:
`OpenDoor()` derives from a different call path then `SyncOpL*Door()`, but was the only meaningful difference. `_oAnimFrame` cannot be shared as it isn't idempotent an the other path already starts with the correct frame.